src/main/java/com/example/DoughnutShop.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+public class DoughnutShop {
+
+    public String prepareDoughnut(String doughnutType) {
+        String preparedDoughnut;
```

Comment:
Could be worth creating an enum for doughnutType

Reply:
Noice point 💪